app/models/eventCohort.py

```diff
@@ -0,0 +1,15 @@
+from datetime import datetime
+from app.models import*
+from app.models.event import Event
+from app.models.bonnerCohort import BonnerCohort
+
+class EventCohort(baseModel):
+    event = ForeignKeyField(Event)
+    year = IntegerField()  
+    invited = BooleanField(default=False)
```

Comment:
I don't mind this, but I would say we probably don't need the boolean. If they are put in this table we can infer they have been invited. Unless there is something else you were hoping to use this field for.